src/ansible_builder/containerfile.py

```diff
@@ -94,7 +94,8 @@ def prepare(self) -> None:
                 self.steps.append('RUN /output/scripts/pip_install $PYCMD')
 
             if self.definition.ansible_ref_install_list:
-                self.steps.append('RUN $PYCMD -m pip install --no-cache-dir $ANSIBLE_INSTALL_REFS')
+                self.steps.append(
+                    'RUN --mount=type=cache,dst=/root/.cache/pip $PYCMD -m pip install -q $ANSIBLE_INSTALL_REFS')
```

Comment:
Using `-q` hides some install info that could be useful in debugging user issues. I'd prefer to not use this.

I did notice that it did subtract about 1.5 seconds on a very simple build, and the resulting image sizes did not seem to change by any significant amount.